src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery;
+        for (int i = 0; i < 3; i++) {
+            lottery = new Lottery();
+            System.out.println(lottery.getRandomBall());
+        }
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You can create a single instance of Lottery before the loop and use it within the loop to get random balls.